docs/src/components/footer/EmailSubscribe.tsx

```diff
@@ -109,23 +108,39 @@ export default function EmailSubscribe({ sx }: { sx?: SxProps<Theme> }) {
           onChange={(event) => setForm({ email: event.target.value, status: 'initial' })}
           inputProps={{ required: true }}
           sx={{
+            minWidth: 220,
             bgcolor: (theme) =>
+              theme.palette.mode === 'dark' ? theme.palette.primaryDark[900] : '#fff',
+            boxShadow: (theme) =>
               theme.palette.mode === 'dark'
-                ? theme.palette.primaryDark[900]
-                : theme.palette.grey[100],
+                ? '0 1px 2px 0 rgba(0 0 0 / 1)'
+                : '0 1px 2px 0 rgba(0 0 0 / 0.1)',
+            borderRadius: 1,
+            border: '1px solid',
+            borderColor: (theme) =>
+              theme.palette.mode === 'dark'
+                ? theme.palette.primaryDark[500]
+                : theme.palette.grey[300],
             px: 1,
             py: 0.5,
             typography: 'body2',
-            flexGrow: 1,
-            minWidth: 200,
+
+            '&:hover': {
+              borderColor: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[300]
+                  : theme.palette.grey[500],
+              boxShadow: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? '0 1px 2px 0 rgba(0 0 0 / 1)'
+                  : '0 1px 2px 0 rgba(0 0 0 / 0.15)',
+            },
+
             '&:focus': {
-              outline: (theme) =>
-                `2px solid ${
-                  theme.palette.mode === 'dark'
-                    ? theme.palette.primary[400]
-                    : theme.palette.primary[200]
-                }`,
-              outlineOffset: '2px',
+              borderColor: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[300]
+                  : theme.palette.primary[500],
```

Comment:
@siriwatknp for some reason this is not being picked up, do you know why?! Tried targeting specific classes but even then 🤔

Reply:
It's because the focus pseudo-class does nothing when applied to a `<div>`.

<img width="251" alt="Screenshot 2022-07-24 at 21 31 00" src="https://user-images.githubusercontent.com/3165635/180662879-24b6b261-31b1-43b8-87ad-0dfd148a2c8c.png">

Reply:
Updated! you have to target `&.${inputClasses.focused}`.

Reply:
Right, this works as well 👍, it what we use in the source of the published components